core-blocks/gallery/index.js

```diff
@@ -207,9 +199,9 @@ export const settings = {
 			},
 
 			save( { attributes } ) {
-				const { images, columns = defaultColumnsNumber( attributes ), align, imageCrop, linkTo } = attributes;
+				const { images, columns = defaultColumnsNumber( attributes ), imageCrop, linkTo } = attributes;
```

Comment:
It shouldn't be removed from the deprecated version because it will break it. In addition:
```js
align: {
	type: 'string',
	default: 'none',
},
```
should be added to the list of attributes.

Reply:
Addressed in [this](https://github.com/WordPress/gutenberg/pull/7100/commits/9374ed318ead245d1b3ba974307c1c7a62bc7096) commit. Thank you ❤️